sunpy/instr/goes.py

```diff
@@ -276,7 +380,7 @@ def goes_chianti_tem(longflux, shortflux, satellite=8,
     # GOES 6 long channel flux before 1983-Jun-28 must be corrected by a
     # factor of 4.43/5.32
     if date < datetime.datetime(1983, 06, 28) and satellite == 6:
-        longflux_corrected = longflux * (4.43/5.32)
+        longflux_corrected = longflux*(4.43/5.32)
```

Comment:
They come from an incorrect rescaling of the publicly available GOES data.  I think it was that later GOES data was rescaled to be consistent with earlier data.  But then the new data turned out to be correct.  Or something like that.  The GOES data was never rescaled as people were already rescaling it themselves or something.  Again, it's one of those things you just have to know. But I don't think it's actually SSW's fault.